[BUG] @prizm-ui/components - Отсутствие зависимости lodash-es при попытке запустить проект.

### Библиотека

_Выберите нужное отметив галочку [x]_

- [ ] `@prizm-ui/core`
- [x] `@prizm-ui/components`
- [ ] `@prizm-ui/install`
- [ ] `@prizm-ui/icons`
- [ ] `@prizm-ui/flag-icons`
- [ ] `@prizm-ui/theme`
- [ ] `@prizm-ui/charts`
- [ ] `@prizm-ui/ast`
- [ ] `@prizm-ui/nx-plugin`
-

### Компонент

@prizm-ui/components

### Версия

Операционная ситема:

`Windows 11, etc`

Браузер:

`Google Chrome Версия 119.0.6045.106 (Официальная сборка), (x86_64)`

NodeJS:

```
➜ node --version
v16.13.0
```

Dependencies:

```
➜ npm ls @prizm-ui/components

@prizm-ui/components 1.8.0
```

```
➜ npm ls @angular/core

@prizm-ui/components@0.0.2
├─┬ @angular/animations@12.2.12
│ └── @angular/core@12.2.12 deduped
├─┬ @angular/cdk@12.2.13
│ └── @angular/core@12.2.12 deduped
├─┬ @angular/common@12.2.12
│ └── @angular/core@12.2.12 deduped
├── @angular/core@12.2.12
├─┬ @angular/forms@12.2.12
│ └── @angular/core@12.2.12 deduped
├─┬ @angular/platform-browser-dynamic@12.2.12
│ └── @angular/core@12.2.12 deduped
├─┬ @angular/platform-browser@12.2.12
│ └── @angular/core@12.2.12 deduped
├─┬ @angular/router@12.2.12
│ └── @angular/core@12.2.12 deduped
├─┬ @storybook/angular@6.3.12
│ └── @angular/core@12.2.12 deduped
├─┬ jest-preset-angular@10.0.1
│ └── @angular/core@12.2.12 deduped
└─┬ primeng@12.2.2
  └── @angular/core@12.2.12 deduped


```

### Скрин / Видео проблемы

`./node_modules/@prizm-ui/components/fesm2020/prizm-ui-components.mjs:32:0-44 - Error: Module not found: Error: Can't resolve 'lodash-es/cloneDeep' in 'C:\projects\zui-pkg-report\node_modules\@prizm-ui\components\fesm2020'`


### Описание проблемы

1. Обновил @prizm-ui/components до версии 1.8.0
2. Установил зависимости
Результат: При попытке запустить проект получаю ошибку
`./node_modules/@prizm-ui/components/fesm2020/prizm-ui-components.mjs:32:0-44 - Error: Module not found: Error: Can't resolve 'lodash-es/cloneDeep' in 'C:\projects\zui-pkg-report\node_modules\@prizm-ui\components\fesm2020'`
Данная зависимость нигде не указана как необходимая.

--- a/libs/charts/base/package.json
+++ b/libs/charts/base/package.json
@@ -8,6 +8,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^16.1.8",
     "@angular/core": "^16.1.8",
     "@prizm-ui/theme": "^3.4.0",

--- a/libs/charts/base/package.json.ng14
+++ b/libs/charts/base/package.json.ng14
@@ -8,6 +8,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^14.2.12",
     "@angular/core": "^14.2.12",
     "@prizm-ui/theme": "^1.8.0",

--- a/libs/charts/base/package.json.ng15
+++ b/libs/charts/base/package.json.ng15
@@ -8,6 +8,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^15.2.0",
     "@angular/core": "^15.2.0",
     "@prizm-ui/theme": "^2.3.0",

--- a/libs/charts/base/package.json.ng16
+++ b/libs/charts/base/package.json.ng16
@@ -8,6 +8,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^16.1.8",
     "@angular/core": "^16.1.8",
     "@prizm-ui/theme": "^3.4.0",

--- a/libs/components/package.json
+++ b/libs/components/package.json
@@ -9,6 +9,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^16.1.8",
     "@angular/core": "^16.1.8",
     "@angular/cdk": "^16.1.7",

--- a/libs/components/package.json.ng14
+++ b/libs/components/package.json.ng14
@@ -9,6 +9,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^14.2.12",
     "@angular/core": "^14.2.12",
     "ngx-mask": "14.2.4",

--- a/libs/components/package.json.ng15
+++ b/libs/components/package.json.ng15
@@ -9,6 +9,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^15.2.0",
     "@angular/core": "^15.2.0",
     "@angular/cdk": "^15.2.0",

--- a/libs/components/package.json.ng16
+++ b/libs/components/package.json.ng16
@@ -9,6 +9,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "peerDependencies": {
+    "lodash-es": "^4.17.21",
     "@angular/common": "^16.1.8",
     "@angular/core": "^16.1.8",
     "@angular/cdk": "^16.1.7",

